packages/react-server/core/renderMiddleware.js

```diff
@@ -1070,6 +1071,17 @@ function getNonInternalConfigs() {
 	return nonInternal;
 }
 
+function getDeviceType(req) {
+	var md = new MobileDetect(req.get('user-agent'));
+	var types = [ 'phone', 'tablet', 'mobile' ];
```

Comment:
This still includes `mobile`, but the PR comments say we're eliminating that. Can it be removed here?

Reply:
Thank goodness you're reviewing this carefully. 😳

Reply:
For once ;)